docs/sphinx/source/conf.py

```diff
@@ -0,0 +1,54 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'Bathymetric Attributed Grid (BAG)'
+copyright = '2022, The Open Navigation Surface Project'
+author = 'The Open Navigation Surface Project'
+release = '2.0.1'
```

Comment:
One more place to update at each new release. Should we add all of them in a how-to-release doc?

Reply:
Good catch, thanks. Addressed [here](https://github.com/OpenNavigationSurface/BAG/commit/a529bda216220fcb2d553db3901f920c1f3842ad).